scripts/version.js

```diff
@@ -91,6 +93,30 @@ async function updatePackageConfig(packageName, transform) {
   await jsonfile.writeFile(file, json, { spaces: 2 });
 }
 
+/**
+ * @param {string} example
+ * @param {(json: string) => any} transform
+ */
+ async function updateExamplesPackageConfig(example, transform) {
+  let file = packageJson(example, "examples");
+  let json = await jsonfile.readFile(file);
+  transform(json);
+  await jsonfile.writeFile(file, json, { spaces: 2 });
+}
+
+/**
+*
+* @param {string} example
+* @param {string} version
+*/
+async function updateExampleReadmeUrl(example, version) {
+  let filePath = path.join(rootDir, "examples", example, "README.md")
+  let fileBuffer = await fs.readFile(filePath)
+  let file = fileBuffer.toString().replace(`https://stackblitz.com/github/remix-run/react-router/tree/dev/examples/${example}`, `https://stackblitz.com/github/remix-run/react-router/tree/${version}/examples/${example}`)
```

Comment:
I think we might need to adjust this a little for subsequent publishes. After the first publish the link won't be pointing to `tree/dev` anymore. Instead it will be pointing to whatever that version was (e.g. `tree/v6.0.0`). So maybe we can do something like

```tsx
let file = fileBuffer.toString().replace(
  /https:\/\/stackblitz\.com\/github\/remix-run\/react-router\/tree\/[^\/+]\/examples\/\w+/,
  `https://stackblitz.com/github/remix-run/react-router/tree/${version}/examples/${example}`
);
```

Does that make sense?

Reply:
ah yea! i was thinking it was going to stay at dev for some reason and only the tags would be adjusted 😅